lib/Entity/User.php

```diff
@@ -261,6 +261,8 @@ class User implements \JsonSerializable
      */
     public $twoFactorRecoveryCodes = [];
 
+    public $showContentFrom = 1;
```

Comment:
Can we have a description of this property here as its not immediately obvious what it means from the var name?